static/js/components/EmploymentTab.js

```diff
@@ -261,9 +261,11 @@ class EmploymentTab extends ProfileTab {
         <Grid className="profile-tab-grid no-border">
           <Dialog
             open={workDialogVisibility}
-            className="profile-employment-dialog"
+            className="dashboard-dialog"
             onRequestClose={this.closeWorkDialog}
-            actions={actions}>
+            actions={actions}
+            autoScrollBodyContent={true}
```

Comment:
oh snazzy! I thought we've have to muck around with `overflow: scroll` and all that.